src/chrome/content/rules/film.ru.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="film.ru">
+	<target host="film.ru" />
+	<target host="www.film.ru" />
+	<!-- <target host="entourage.film.ru" /> --><!-- ERROR (35, schannel: next InitializeSecurityContext failed: SEC_E_UNTRUSTED_ROOT (0x80090325) -     ,   .) -->
+	<target host="forum.film.ru" />
```

Comment:
https://forum.film.ru and http://forum.film.ru serve different content, and from a quick look at it, this one needs to go completely. LGTM otherwise.